src/index.stories.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
+// tslint:disable-next-line:no-implicit-dependencies
```

Comment:
This is only used in development mode and for building the storybook.
But the linter thinks it would be a runtime depdency.